test/unit/argument_parser/format_parse_validators_test.cpp

```diff
@@ -919,21 +1003,26 @@ TEST(validator_test, regex_validator_success)
     option_vector.clear();
     const char * argv7[] = {"./argument_parser_test", "-h"};
     seqan3::argument_parser parser7{"test_parser", 2, argv7, false};
+    test_accessor::set_terminal_width(parser7, 80);
     parser7.add_option(option_vector, 's', "string-option", "desc",
                        seqan3::option_spec::DEFAULT, email_vector_validator);
 
     option_vector.clear();
     testing::internal::CaptureStdout();
     EXPECT_EXIT(parser7.parse(), ::testing::ExitedWithCode(EXIT_SUCCESS), "");
     std::string my_stdout = testing::internal::GetCapturedStdout();
-    std::string expected = std::string("test_parser"
-                           "===========" +
+    std::string expected = std::string("test_parser\n"
+                           "===========\n"
+                           "\n" +
                            basic_options_str +
-                           "    -s, --string-option (List of std::string's)"
-                           "          desc Default: []. Value must match the pattern '[a-zA-Z]+@[a-zA-Z]+\\.com'." +
+                           "\n"
+                           "  \n"
+                           "    -s, --string-option (List of std::string's)\n"
```

Comment:
the same

Reply:
I added a Card https://github.com/seqan/seqan3/projects/7#card-35506628 for these little wording mistakes.